File: AppCommonModule/src/main/java/com/lightheart/common/CommonMainActivity.java
package com.lightheart.common;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;

/**
 * @author liaohui
 * @date 2019/7/7
 */
public class CommonMainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.lib_common_activity_main);
    }
}

File: AppCommonModule/src/main/java/com/lightheart/common/CommonMainActivity.kt
package com.lightheart.common

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import kotlinx.android.synthetic.main.lib_common_activity_main.*

class CommonMainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.lib_common_activity_main)
        test_view.setOnClickListener { Toast.makeText(this, "xixi", Toast.LENGTH_SHORT).show() }
    }
}
